docs/release-notes.rst

```diff
@@ -6,6 +6,53 @@
  Release Notes
 ###############
 
+**************
+ Version 0.30
+**************
+
+Version 0.30.0
+==============
+
+**Release Date:** April 04, 2024
+
+**Breaking Changes**
+
+-  API: :class:`~determined.pytorch.Trainer` no longer supports the ``Trainer.configure_profiler``
+   option. Profiling is now enabled through the ``Trainer.fit(profiling_enabled=True)`` call.
+
+-  Database migration: system metrics collected by the Determined profiler are now stored in the
+   generic ``metrics`` table. This requires a few schema changes to the ``metrics`` table that will
+   be run during migrations.
+
+.. important::
+
+   This migration will be more time-consuming for deployments with a large amount of metrics stored.
+
+**New Features**
+
+-  Core API: the Determined profiler is now accessible from the Core API. It collects system
+   metrics, which can be viewed in the Web UI under the experiment's "Profiler" tab. See the
+   :ref:`Core API guide <core-profiler>` for details.
```

Comment:
Does this page exist?

Reply:
Yes! It points here: model-dev-guide/api-guides/apis-howto/api-core-ug.html#core-profiler